R/get_ref.R

```diff
@@ -150,8 +150,16 @@ get_ref_min.remote_ref_github <- function(remote_ref, op = "", op_ver = "") {
     }
   }
 
-  new_ref <- sprintf("%s=%s/%s%s", remote_ref$package, remote_ref$username, remote_ref$repo, ref_suffix) # @TODO
-  pkgdepends::parse_pkg_ref(new_ref)
+  new_ref <- sprintf(
+    "%s=%s/%s%s",
+    remote_ref$package,
+    remote_ref$username,
+    remote_ref$repo,
+    ref_suffix
+  )
+  result <- pkgdepends::parse_pkg_ref(new_ref)
+
+  result
```

Comment:
`result` seems to be not needed

```suggestion
  pkgdepends::parse_pkg_ref(new_ref)
```